inc/Engine/Optimization/CSSTrait.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace WP_Rocket\Engine\Optimization;
+
+trait CSSTrait {
+
+	/**
+	 * Applies font-display:swap to all font-family rules without a previously set font-display property.
+	 *
+	 * @since 3.7
+	 *
+	 * @param string $css_file_content CSS file content to modify.
+	 *
+	 * @return string Modified CSS content.
+	 */
+	private function apply_font_display_swap( $css_file_content ) {
+		$css_file_content = (string) $css_file_content;
+
+		$replacement = preg_replace_callback(
+			'/(?:@font-face)\s*{([^}]+)}/',
+			function ( $matches ) {
+				$matches[1] = ( false !== strpos( $matches[1], 'font-display' ) )
+					? $matches[1]
+					: 'font-display:swap;' . $matches[1];
+
+				return '@font-face{' . $matches[1] . '}';
```

Comment:
Instead of using a ternary operator, what do you think of this?

```
if ( false === strpos( $matches[value], 'font-display' ) ) {
    return $matches[0];
}

$swap = "font-display:swap;{$matches['value']}";

return str_replace( $matches['value'], $swap, $matches[0] );
```

The end result is the same, but you have an early bail-out in case the property is already set, and it's also easier to read

Reply:
Still needs to be `false !== strpos()`, but yeah!